dev/breeze/src/airflow_breeze/ci/build_params.py

```diff
@@ -0,0 +1,121 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+from dataclasses import dataclass
+from datetime import datetime
+
+from airflow_breeze.breeze import get_airflow_sources_root
+from airflow_breeze.utils import run_command
+
+
+@dataclass
+class BuildParams:
+    # To construct ci_image_name
```

Comment:
@potiuk I have set all the default value for Params as default value for variable. Do you think it is right to do this way?

Reply:
Absolutely.